### DISCUSSION TITLE
Cronjob (URL-based CRON) is running but installation site does not change to benchmark

### DISCUSSION THREAD

**Discussion by Martin511** [akeeba/panopticon#687] [Q&A (Community Support)] [answered]
Hello.
I´m installing Panopticon and I have set an URL-based CRON.
The CRON is running every minute but the installation site does not change to "benchmark" as described in the installation process.
What can I do now?

**Comment by nikosdion** [comment]
It will automatically switch to benchmark _as long as_ the CRON job is executing successfully.

The way this works is that every time you run a CRON job it marks the time it started executing. When this time is within the last minute, the page switches to benchmark (it checks for that timestamp every few seconds).

Have you checked your server access log to confirm that the CRON job is really running?

**Comment by Martin511** [comment]
As I have no access to the server logs, I will have to ask my hoster tomorrow.
But if I logout of Panopticon and log in again - and I am not forced to set the cronjob - then  I think it is running, right?

**Reply by nikosdion** [reply, marked as answer]
If it's not showing you a banner that the CRON jobs are not running then, yup, it's running.
